app/views/subscription_mailer/_summary_detail.html.haml

```diff
@@ -6,7 +6,7 @@
 
   - separator = messages.values.any? ? ": " : ", "
   - orders.each_with_index do |order, i|
-    %a{ href: order_url(order) }>= order.number
+    %a{ href: spree.edit_admin_order_url(order) }>= order.number
```

Comment:
Just for context, this partial is used in placement_summary_email and confirmation_summary_email which are both sent to the shop's contact email.

I just wanted to check that none of this is sent to customers. It looks fine.

Reply:
Thanks for checking this. Perhaps in the future we should arrange mailer views according to their audience, with an `admin/` folder for those that are sent to admin users.